projects/rockdaboot.github.io/libpsl/package.yml

```diff
@@ -0,0 +1,36 @@
+distributable:
+  url: https://github.com/rockdaboot/libpsl/releases/download/{{version}}/libpsl-{{version}}.tar.gz
+  strip-components: 1
+versions:
+  github: rockdaboot/libpsl
+dependencies:
+  unicode.org: '*'
+build:
+  dependencies:
+    mesonbuild.com: '*'
+    ninja-build.org: '*'
+    freedesktop.org/pkg-config: '*'
+    python.org: ~3.11
+    linux:
+      gnu.org/gcc: '*'
+  script:
+    - meson setup build $MESON_ARGS
+    - meson compile -C build
+    - meson install -C build
+  env:
+    linux:
+      LDFLAGS: -fPIC
```

Comment:
stuff like this should also have a comment explaining why pls

Reply:
> stuff like this should also have a comment explaining why pls

Oh okay 👌🏻 
In this case, this solves the error:
`undefined reference to 'main'`

Reply:
I will comment this from now